src/main/java/seedu/address/model/person/Person.java

```diff
@@ -45,38 +46,80 @@ public Person(Name name, Map<String, Phone> numbers, Map<String, Email> emails,
         this.tags.addAll(tags);
     }
 
+    /**
+     * Returns an empty person with nothing values.
+     *
+     * @return an empty person with nothing values
+     */
+    public static Person getEmptyPerson() {
+        return new Person(new Name("No name"), new HashMap<>(), new HashMap<>(), new HashMap<>(),
+                new Company("No Company"), new JobTitle("No JobTitle"), new HashSet<>(), new HashSet<>());
+    }
+
     public Name getName() {
         return name;
     }
 
+    public void setName(Name name) {
+        this.name = name;
+    }
+
     public Map<String, Phone> getNumbers() {
-        return numbers;
+        return Collections.unmodifiableMap(numbers);
```

Comment:
I was reading online that there is an unmodifiableMap but there is also an immutableMap, and that unmodifiableMap can still be changed but immutableMap cannot. Maybe worth seeing if immutableMap is better, I'm not 100% sure.